tests/lib/rules/no-implicit-globals.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * @fileoverview Tests for no-implicit-globals rule.
+ * @author Joshua Peek
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/rules/no-implicit-globals"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester();
+ruleTester.run("no-implicit-globals", rule, {
+    valid: [
+        {
+            code: "window.foo = 1;"
+        },
+        {
+            code: "window.foo = function() {};"
+        },
+        {
+            code: "window.foo = function foo() {};"
```

Comment:
I *think* this is supposed to be okay. But I remember some old IE cases where the function name here would leak.